dace/sdfg/validation.py

```diff
@@ -160,7 +218,29 @@ def validate_state(state: 'dace.sdfg.SDFGState',
     for nid, node in enumerate(state.nodes()):
         # Node validation
         try:
-            node.validate(sdfg, state)
+            if isinstance(node, nd.NestedSDFG):
+                #Add constant's defined by unrolled maps temporary as sdfg-level
+                #constants with a dummy value and remove the corresponding symbol-bindings
+                #on the nsdfg
```

Comment:
Would it be an option to add some dict argument to the validate call (default None), that can contain options for the validation? Then one could simply pass the scope local constants to nested sdfg's via this dict, instead of somehow adding them to the sdfg. Not so nice to change the validate interface just for this, but probably still nicer than modifying the SDFG?

Reply:
@tbennun Do you have an opinion here?